recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -123,10 +124,12 @@ do_install () {
 	install -m 0644 ${S}/uixml/* ${D}${uixmldir}/
 
 	# Common interface for system settings (soft dip switches, etc.)
-	install -d -m 0775 ${D}${settingsdatadir}/
-	chown ${LVRT_USER}:${LVRT_GROUP} ${D}${settingsdatadir}/
+	install -d -m 0775 -o ${LVRT_USER} -g ${LVRT_GROUP} ${D}${settingsdatadir}/
```

Comment:
I don't think this directory needs to be set ${LVRT_USER}:${LVRT_GROUP} at all-- it's package-local to us, and I don't believe lvuser ever needs to modify the contents.

Reply:
I'm honestly fine either way, but the sumo version was lvuser:ni and the original bug was that it had changed in hardknott.  I'm not sure what other packages get installed and need to read/write files in this directory, but if you're sure that 0:0 would work, I can change it.  The question is ... are you sure?

Reply:
So there are two directories in play here.

    settingsdatadir = "${datadir}/${BPN}/systemsettings"
    systemsettingsdir = "${localstatedir}/local/natinst/systemsettings"

the first one did not exist in sumo, it was added in 633c3f2e072e.

(the latter one did and should probably be lvuser:ni for consistency)

Reply:
Oh sorry missed your point on that one.  Yea I'm totally fine removing the lvuser:ni on settingsdatadir.